File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let answer = true;
    const sets = new Set(nums)
    console.log(nums.length);
    console.log(sets.size);
    if(nums.length == sets.size){
        answer = false;
        return answer;
    }
    return answer;
    
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
    #Convert param(nums)list to set to remove the duplicate
    #Convert set to list again
        my_list = list(set(nums))
    #If nums doesn't have duplication, the my_list and nums leggth will be same. If they are same, return false.
        if(len(nums)==len(my_list)):
            return False
    #Otherwise, return true 
        return True
    